src/views/virtualmachines/creation-wizard-new/hooks/useVMGenerationNavClick.ts

```
@@ -0,0 +1,48 @@
+} from '@virtualmachines/creation-wizard-new/utils/constants';
+import {
+  isInstanceTypeCreationMethod,
+  isTemplateCreationMethod,
+} from '@virtualmachines/creation-wizard-new/utils/utils';
+
+import useGenerateVM from '../steps/InstanceTypesSteps/hooks/useGenerateVM/useGenerateVM';
+import { WizardStepNavItemConfig } from '../utils/types';
+
+const useVMGenerationNavClick = (
+  creationMethod: VMCreationMethod,
+): WizardStepNavItemConfig => {
```

ESLint

[error] Replace `⏎··creationMethod:·VMCreationMethod,⏎` with `creationMethod:·VMCreationMethod`

(prettier/prettier)